internal/api/mocks/schema_service.go

```diff
@@ -168,7 +213,15 @@ func (_m *SchemaService) GetMetadata(ctx context.Context, namespace string, sche
 func (_m *SchemaService) List(ctx context.Context, namespaceID string) ([]schema.Schema, error) {
 	ret := _m.Called(ctx, namespaceID)
 
+	if len(ret) == 0 {
+		panic("no return value specified for List")
+	}
+
```

Comment:
can we return an error instead of panic, since this method returns an error anyways?